workflow_plandefinition/models/workflow_plan_definition_action.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models, api, exceptions
+
+
+class PlanDefinitionAction(models.Model):
+    _name = 'workflow.plan.definition.action'
+    _description = 'Medical Plan Definition Action'
+    _inherit = 'mail.thread'
+    _parent_name = 'parent_id'
```

Comment:
Hmmmm is this an interface that we can use in child models?

Reply:
As far as I know this defines the name by which you refer to the parent. As by default it is 'parent_id' in this case it is not necessary. Is that what you mean?